backend/src/main/java/stand/lemonade/service/WalletServiceImpl.java

```
@@ -35,7 +35,7 @@ public WalletServiceImpl() {
 	
 	@Transactional
 	@Override
-	public List<WalletModel> getAvailableWallets() {
+	public List<WalletModel> getAvailableWallets() throws LemonadeException {
```

why would this method need to throw an exception?